doc/appendix/faq.tex

```diff
@@ -224,4 +224,48 @@ \subsubsection{Error in job submit request.}
 	sacct -j 1234
 \end{verbatim}
 
-\noindent A summary of the reasons can be obtained via the \tool{squeue} command.
\ No newline at end of file
+\noindent A summary of the reasons can be obtained via the \tool{squeue} command.
+
+% B.6 How to Handle /tmp Overflow on Speed Nodes?
+% -------------------------------------------------------------
+\subsection{How to Handle \tool{/tmp} Overflow on Speed Nodes?}
```

Comment:
@salhanyf -- please do not use `\tool{}` for paths; `\tool{}` is for executable commands. For now please use `\texttt{/tmp}` for all instances.